pkg/util/github/github.go

```diff
@@ -157,6 +159,17 @@ func DownloadOpenShiftReleaseBinary(binaryType OpenShiftBinaryType, osType minis
 				fmt.Println()
 			}()
 		}
+
+		if httpResp.ContentLength < 0 {
+			fmt.Print("   Downloading ")
+			progressDots := progressdots.New()
+			progressDots.Start()
+			defer func() {
+				progressDots.Stop()
+				fmt.Print(" OK")
+				fmt.Println()
+			}()
+		}
```

Comment:
With this change 

```
-- Downloading OpenShift binary 'oc' version 'v3.10.0'
   Downloading ............-- Downloading OpenShift v3.10.0 checksums ... OK OK => broken UX
-- Checking if provided oc flags are supported ... OK
-- Starting the OpenShift cluster using 'kvm' hypervisor ...
-- Minishift VM will be configured with ...
   Memory:    4 GB
   vCPUs :    2
   Disk size: 20 GB

   Downloading ISO 'https://github.com/minishift/minishift-centos-iso/releases/download/v1.12.0/minishift-centos7.iso'
=> here no progress dots or progress bar :(
```

Reply:
Broken UI I knew. We write chcksum before finishing. Hadnt checked ISO yet